vocab/properties.json

```diff
@@ -639,14 +613,14 @@
         ]
     },
     "fileExtension": {
-        "description": null,
+        "description": "String of characters attached as suffix to the names of files of a particular format.",
         "name": "File extension",
         "schemas": [
             "core/schemas/v3/data/contentType.schema.tpl.json"
         ]
     },
     "format": {
-        "description": null,
+        "description": "Method of organizing data.",
```

Comment:
Also here I've sticked close to https://www.merriam-webster.com/dictionary/format (third meaning). Note also, stating the general meaning is fine if we also use this word in the general sense. If we use the word in a more specific meaning we should choose the description accordingly. It is our namespace description therefore we can be specific in what we mean. Nonetheless in some cases we want the general meaning because we do use or may want to use the word in the general sense.